Fotogjengen/hilfling-frontend
# Feature/endre/altert deletes feedback archive boss

- made popup delete box when trying to delete a album,
- made a alert when you delete a album

testing:
1. run the backend,
2. go to /intern/arkivsjef
3. a popup dialog box should appear when you try deleting an album
4. an alert on top opp the screen will appear when you press delete in the dialog box
